build-tools/automation/yaml-templates/start-emulator.yaml

```diff
@@ -0,0 +1,18 @@
+parameters:
+  installImage: false    # 'true' to create a new specific AVD image emulator, if 'true', other 'install*' parameters must be provided
+  installDeviceName:     # Device name to install, like 'wear_square'
+  installAvdApiLevel:    # Device API level to install, like '30'
+  installAvdAbi:         # Device ABI to install, like 'x86'
+  installAvdType:        # Device AVD to install, like 'android-wear'
+
+steps:
+# This uses our in-tree 'dotnet' because the system one may not have the 'android' workload installed.
+- template: run-dotnet-preview.yaml
+  parameters:
+    displayName: Start emulator
+    continueOnError: false
+    project: tests/Mono.Android-Tests/Runtime-Microsoft.Android.Sdk/Mono.Android.NET-Tests.csproj
+    ${{ if eq(parameters.installImage, true) }}:
+      arguments: -c $(XA.Build.Configuration) -t:"InstallAvdImage;AcquireAndroidTarget" -p:TestDeviceName=${{ parameters.installDeviceName }} -p:TestAvdApiLevel=${{ parameters.installAvdApiLevel }} -p:TestAvdAbi=${{ parameters.installAvdAbi }} -p:TestAvdType=${{ parameters.installAvdType }} -bl:$(System.DefaultWorkingDirectory)/bin/Test$(XA.Build.Configuration)/install-emulator-${{ parameters.installAvdApiLevel }}.binlog
+    ${{ else }}:
+      arguments: -c $(XA.Build.Configuration) -t:AcquireAndroidTarget -bl:$(System.DefaultWorkingDirectory)/bin/Test$(XA.Build.Configuration)/start-emulator.binlog
```

Comment:
Could we use one YAML template for starting and stopping the emulator? I think we could combine these and add an extra parameter that determines whether we want to start or stop it.

Reply:
Updated.